.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,45 @@
+name: e2e
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+  schedule:
+  - cron: '0 0 * * *' # daily to pick up releases
+jobs:
+  kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.17'
+    - name: Install kind
+      run: |
+        curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
+        chmod +x kind
+        sudo mv kind /bin/
+    - name: Create kind cluster
+      run: |
+        kind create cluster
+        kind export kubeconfig
+    - name: Run Combo on kind
+      run: make run-local
+  minikube:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.17'
+    - name: Install minikube
+      run: |
+        sudo apt-get install conntrack
+        curl -sLo minikube "$(curl -sL https://api.github.com/repos/kubernetes/minikube/releases/latest | jq -r '[.assets[] | select(.name == "minikube-linux-amd64")] | first | .browser_download_url')"
+        chmod +x minikube
+        sudo mv minikube /bin/
+    - name: Setup minikube
+      run: |
+        minikube config set vm-driver docker
+        minikube start
+    - name: Run Combo on minikube
+      run: make run-local RUN_LOCAL_TYPE=MINIKUBE
```

Comment:
I personally think it's overkill to run e2e tests on both kind and minikube and leads to longer e2e times. If we're interested in ensuring compatibility with minikube, maybe make a dedicated action that runs when PRs land in master?

Reply:
So would the `minikube` test only run when we merge into main, in this case?

Reply:
Yeah that's what I had in mind, but I also think you could just drop minikube support entirely and only test on kind clusters, especially since this project is in greenfield stages.

Reply:
Let's go with that for now, I think we will eventually want to guarantee Minikube functionality though - right?